External/FEXCore/Source/Common/Paths.cpp

```diff
@@ -9,8 +10,8 @@
 #include <unistd.h>
 
 namespace FEXCore::Paths {
-  std::unique_ptr<std::string> CachePath;
-  std::unique_ptr<std::string> EntryCache;
+  std::unique_ptr<fextl::string> CachePath;
```

Comment:
A little unrelated, but wouldn't these unique_ptr's be able to be `std::optional` instances instead? Seems like it would get rid of the need to heap allocate the strings themselves, and would just require handling the empty cases in `GetCachePath` and `GetEntryCachePath`

Reply:
Converted over in the followup PR to ensure this doesn't change behaviour here.
There's a bunch of surrounding changes that makes changing behaviour beyond names a pain.